plugins/SitesManager/API.php

```diff
@@ -1346,6 +1358,68 @@ public function setDefaultTimezone($defaultTimezone)
         return true;
     }
 
+    /**
+     * Sets global query parameter exclusion based on the specified exclusion type.
+     *
+     * @param string $exclusionType The type of query param exclusion, must be of the following:
+     *  - no_exclusions
+     *  - common_pii_exclusions
+     *  - custom_exclusions
+     * @param string|null $queryParamsToExclude (Optional) List of query parameters to exclude when $exclusionType is 'custom'.
+     *                                         Ignored if $exclusionType is not 'custom'.
+     * @return void
+     * @throws InvalidArgumentException
```

Comment:
```suggestion
     * @throws Exception
```

With `Exception` being thrown explicitly, this also covers the validator exception. And with `InvalidArgumentException` missing any namespace information we would document throwing a class that does not exist.

Reply:
Good catch, was a refactor without a good cleanup